csi.proto

```diff
@@ -1303,6 +1312,15 @@ message NodeExpandVolumeRequest {
   // plugin MAY expand the volume to its maximum capacity.
   // This field is OPTIONAL.
   CapacityRange capacity_range = 3;
+
+  // Volume capability describing how the CO intends to use this volume.
+  // This allows SP to determine if volume is being used as a block
+  // device or mounted file system. For example - if volume is being
+  // used as a block device the SP MAY choose to skip expanding the
+  // filesystem. If volume_capability is omitted the SP MAY determine
+  // access_type from given volume_path for the volume and perform
+  // node expansion. This is an OPTIONAL field.
+  VolumeCapability volume_capability = 4;
```

Comment:
for ``` If volume_capability is omitted the SP MAY determine access_type from given volume_path for the volume and perform node expansion```, may be add some more details about how does SP determine this?

Reply:
I am thinking of adding some examples that SP MAY query "volume_path" and determine if given volume_path is a block device or mounted filesystem. But being too specific here requires us bring in operating system and file system specific calls (`Stat`), which may not be suitable for higher level specs.